_projects/iceberg.md

```diff
@@ -6,9 +6,25 @@ abstract: "Coupling open-source image analysis tools with the use of high
 performance and distributed computing (HPDC) for imagery-enabled geoscience
 research." 
 status: active
+logo: iceberg_logo.png
 ---
 
-ICEBERG builds the cyberinfrastructure required to make the most of satellite imagery for geosciences, starting with researchers working in polar areas, and then branching out to the larger community. This cyberinfrastructure brings together cutting edge analytical tools and high performance computing resources to study petabytes of high-resolution imagery. With these tools, the geosciences community can process near-real time datasets of geological and biological importance at the continental scale, establishing common workflows for reproducible, imagery-based science. ICEBERG supports the understanding of the biological, geological, and hydrological functioning of the polar regions at spatial scales beyond the reach of individual PIs. These scientific findings improve our ability to predict the future dynamics of the polar regions, and are critical to the management of Arctic and Antarctic resources.
-
-<a href="https://github.com/iceberg-project"><i class="fa fa-github"></i>GitHub Repositories</a><br>
-<a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1740595"><i class="fa fa-nsf"></i>NSF Award</a>
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/iceberg_logo.png" width="150">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```